packages/lodestar/src/chain/blocks/index.ts

```diff
@@ -114,6 +114,9 @@ export async function processChainSegment(
       ) {
         continue;
       }
+      if (partiallyVerifiedBlock.ignoreIfFinalized && err.type.code == BlockErrorCode.WOULD_REVERT_FINALIZED_SLOT) {
```

Comment:
should all skips (continues) be logged with logger.debug("skipping block ....",{errCode:BlockErrorCode...})
would be use in debug.

very minor" also could move in the same if block (especially if the logger.debug is to be added)

Reply:
Good idea! Could you do a new PR?